silnlp/nmt/runner.py

```diff
@@ -364,3 +365,32 @@ def save_effective_config(self, path: str, training: bool = False) -> None:
         tf.get_logger().setLevel(level)
         with open(path, "w") as file:
             yaml.dump(config, file)
+
+    def evaluate(self, features_file=None, labels_file=None, checkpoint_path=None):
```

Comment:
What is this method being used for?

Reply:
I have no idea how it got in there.  I am removing it.

Reply:
(Well, I added it, but I don't remember doing so, neither is it called or needed).